C#/C#.sublime-syntax

```diff
@@ -432,14 +433,14 @@ contexts:
         - include: interface_declaration
         - include: delegate_declaration
         - include: method_declaration
-        - match: '\{'
-          scope: punctuation.section.block.begin.cs
-          push:
-            - meta_scope: meta.block.cs
-            - match: '\}'
-              scope: punctuation.section.block.end.cs
-              pop: true
-            - include: data_type_body
+        # - match: '\{'
```

Comment:
Can these commented-out lines be ditched now?

Reply:
Yep, done :)